packages/cli/src/report/report-formats.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export enum ReportFormats {
```

Comment:
we can use type instead of enum `export declare type ReportFormats = 'html' | 'json';`